charts/calico/templates/calico-node-rbac.yaml

```diff
@@ -12,8 +12,10 @@ rules:
     resourceNames:
 {{- if eq .Values.network "flannel" }}
       - canal
+      - canal-cni-plugin
```

Comment:
A bit of a nit, but I'd just call this `calico-cni-plugin` since under the covers it's still using the Calico CNI plugin, even for Canal.